src/components/person.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { MyContext } from './state';
+import { Person as PersonType } from '../types';
+
+interface PersonComponentProps {
+  pers: PersonType;
+}
+
+export const Person: React.FC<PersonComponentProps> = ({ pers }) => {
+  const { name, sex, born, fatherName, motherName, slug, died } = pers;
+  const { setSelectedPerson, person } = useContext(MyContext);
+  const location = useLocation();
+
+  const motherPerson = person.find(
+    (perso: PersonType) => perso.name === motherName,
+  );
+  const fatherPerson = person.find(
+    (perso: PersonType) => perso.name === fatherName,
+  );
+
+  const isPersonInUrl = location.pathname.includes(slug);
+
+  const getPeopleClass = () =>
+    classNames({ 'has-background-warning': isPersonInUrl });
```

Comment:
Code quality issue: The `getPeopleClass` function is called every time the component re-renders. You could use `useMemo` hook to memorize the result and only recompute it when `isPersonInUrl` changes.